README.md

```diff
@@ -39,3 +39,13 @@ The project also draws inspiration from:
 
 The app is running in a hobby instance of vercel:
 [Production instance](https://voice-trainer-kg8920zu9-pisti2010.vercel.app/)
+
+
+## Fernando-pr - Branch push troubleshooting
```

Comment:
No need to specify the branch adding the change. Git is doing that work for us through the git history.